dependencies/influxdb/http.go

```diff
@@ -441,3 +457,59 @@ func (h seriesCardinalityHttpReader) Read(ctx context.Context, f func(flux.Table
 	}
 	return h.Query(ctx, f, &file, h.Bounds.Now, mem)
 }
+
+type httpWriter struct {
+	writer      *api.WriteAPIImpl
+	errChan     <-chan error
+	latestError chan error
+}
+
+func NewHttpWriter(writer *api.WriteAPIImpl) (*httpWriter, error) {
```

Comment:
Leave this unexported too. I don't want to create an external api that others can rely on.